src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -556,21 +557,23 @@ public StudyController(StudyService studyService, NetworkStoreService networkSto
             new DiagramParameters(useName, centerLabel, diagonalLabel, topologicalColoring, componentLibrary), substationLayout, nodeUuid));
     }
 
-    @GetMapping(value = "/studies/{studyUuid}/security-analysis/status")
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/security-analysis/status")
     @Operation(summary = "Get the security analysis status on study")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis status"),
             @ApiResponse(responseCode = "404", description = "The security analysis status has not been found")})
-    public Mono<ResponseEntity<String>> getSecurityAnalysisStatus(@Parameter(description = "Study UUID") @PathVariable("studyUuid") UUID studyUuid) {
-        return studyService.getSecurityAnalysisStatus(studyUuid)
+    public Mono<ResponseEntity<String>> getSecurityAnalysisStatus(@Parameter(description = "Study UUID") @PathVariable("studyUuid") UUID studyUuid,
+                                                                  @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        return studyService.getSecurityAnalysisStatus(nodeUuid)
                 .map(result -> ResponseEntity.ok().body(result))
                 .defaultIfEmpty(ResponseEntity.notFound().build());
     }
 
-    @PutMapping(value = "/studies/{studyUuid}/security-analysis/stop")
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/security-analysis/stop")
     @Operation(summary = "stop security analysis on study")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis has been stopped")})
-    public ResponseEntity<Mono<Void>> stopSecurityAnalysis(@Parameter(description = "Study name") @PathVariable("studyUuid") UUID studyUuid) {
-        return ResponseEntity.ok().body(studyService.stopSecurityAnalysis(studyUuid));
+    public ResponseEntity<Mono<Void>> stopSecurityAnalysis(@Parameter(description = "Study name") @PathVariable("studyUuid") UUID studyUuid,
```

Comment:
i think it should be Sudy uuid  for description

Reply:
Corrected